applications/FluidDynamicsApplication/python_scripts/cfd_utility_functions.py

```diff
@@ -0,0 +1,132 @@
+# Importing the Kratos Library
```

Comment:
If the CFDUtilities are implemented as a class with static methods this could be avoided by exposing such class to Python.

Concerning the `Parameters` input, I think it should be implemented also in the C++ level. The `ModelPart` + `Parameters` version of the methods can call the "standard" functions that are already there.